### PR TITLE
fix Fan speed current to show the fan mode instead of wrong %

### PR DESCRIPTION
**Pull request by tizianodeg**
This PR fixes the fan speed current coils to show the fan mode instead of %
 see #164

### CODE REVIEW

**Comment by tizianodeg** [comment]
Not sure how to test with other nibe models. 
I can only test this on F730. Others may return the correct % value...

File: nibe/data/extensions.json [hunk 1]
```diff
@@ -49,6 +49,60 @@
...
+      "f1155_f1255.json",
+      "f1345.json",
+      "f1355.json"
+    ],
+    "data": {
+      "41256": {
+        "unit": null,
+        "mappings": {
+          "0": "Normal",
+          "1": "Fan mode 1",
```
Comment (elupus) [inline nibe/data/extensions.json:68]: Does this match how things like uplink name them? Looks a bit weird with the parameter being called fan "speed", but the values being called fan "modes"

**Comment by yozik04** [comment]
I somehow missed this PR.

File: nibe/data/extensions.json (hunk 1 above)
Comment (tizianodeg) [reply nibe/data/extensions.json:68]: Hey, no nibe uplink shows the % correctly
image

The configuration on my nibe is Mode 2 == 30%
image

Homeassistant shows 2%
image

I have a F730 not updated yet to the newest firmware with the new uplink feature. % where shown correctly with the old nibe uplink integration

I don't know if this behaves the same on other series.. just checked where the coil is defined and changed all of them...

After understanding that this is not the real % but the configured active one i would rather use AirFlow
But that has no unit and so displayed as state ;)
https://github.com/yozik04/nibe/pull/163

**Review by yozik04 — APPROVED** [review]

**Comment by yozik04** [comment]
Thank you!

**Pull request merged**